cuML/CMakeLists.txt

```diff
@@ -98,6 +101,7 @@ set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -gencode arch=compute_${ptx},code=comp
 # - include paths ---------------------------------------------------------------------------------
 
 include_directories(src
+ 		    ${CUDA_INCLUDE_DIR}
```

Comment:
overall I think this variable can be avoided if using `"${CMAKE_CUDA_TOOLKIT_INCLUDE_DIRECTORIES}"`